lib/grpc/GrpcService.ts

```diff
@@ -1,5 +1,7 @@
 /* tslint:disable no-floating-promises no-null-keyword */
-import grpc, { status, ServerWritableStream } from 'grpc';
+import grpc, { ServerWritableStream, status } from 'grpc';
+import * as rxjs from 'rxjs';
```

Comment:
Minor nit: think we can `import { fromEvent } from 'rxjs'` here since it's the only one being used in this file.